src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommandParser.cs

```diff
@@ -19,6 +19,10 @@ public static Command InstallTool()
                     "-g|--global",
                     LocalizableStrings.GlobalOptionDescription,
                     Accept.NoArguments()),
+                Create.Option(
+                    "--bin-path",
+                    "Location of tool shim that is unique for each session",
```

Comment:
pending loc

Reply:
As a user, I will be wondering what a "session" is.

Reply:
@KathleenDollard forget to ask you to check this user facing message

Reply:
shell session? It is a term tho https://askubuntu.com/questions/596420/whats-the-shell-session

Reply:
More like:

"Location of shim to access tool, allowing unique path for each shell session."

I don't love this, I'm still working on it because I hate that we are not explaining that it is a temporary as in vulnerable installation.

Reply:
But why is it temporary and vulnerable? I think we need to stop thinking in time dimension ("sessions") and just worry about space (global location vs. explicit location vs. default local location). If I control where, I can control when to cleanup. "sessions" becomes a pattern (install to local location, use, uninstall from local location), but not a primitive concept to the CLI.

Reply:
@KathleenDollard @nguerrera In implementation it does have nothing to do with sessions. But the goal of CI author is to achieve isolation from different run. Maybe `Location of shim to access tool. Different path can isolate different tool sets.`

Reply:
Do you think it is ok to give a paragraph here? `This is suitable for automation scenario, recommend each automation run to have a unique PATH so there is no interact with different CI run....` The good part is people don't need to look up for document. And bad part is if we are wrong, we stuck with it

Reply:
I'd rather we just say what it does, which is much more general than these scenarios. It seems obvious to me why one *might* pick a unique location. To me, this is like saying `man cp` should say "pick a unique destination if you don't want to share the copied file with other things."